pom.xml

```diff
@@ -61,6 +61,7 @@
                     <artifactSet>
                         <includes>
                             <include>com.github.CodingAir:CodingAPI</include>
+                            <include>org.xerial:sqlite-jdbc</include>
```

Comment:
Hi, thank you for your contribution.

One major issue here: I can't include the sqlite-jdbc driver into the jar. This would result a 8MB huge plugin file.

Reply:
Hmm ok, I didn't realize that was an issue.
Would you prefer If I implemented a mysql connector?

Reply:
Yes, please

Reply:
Sorry for the late response.
I changed it to use a db connection with an hikari datasource.
The jar is around 1.8MB now